manifests/patch/kustomization.yaml

```diff
@@ -39,3 +40,11 @@ patches:
         configmanagement.gke.io/arch: "csmr"
     spec:
       preserveUnknownFields: false
+- patch: |-
+      apiVersion: apiextensions.k8s.io/v1
+      kind: CustomResourceDefinition
+      metadata:
+        name: resourcegroups.kpt.dev
```

Comment:
if we leave out the name with this patch all CRDs?

Reply:
Not sure, I assume it would patch all CRDs that are being input into this kustomization

Reply:
fine for now